.github/workflows/build.yml

```diff
@@ -10,10 +10,10 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v2
       - name: Set up Java 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           java-version: '11'
-          distribution: 'zulu'
+          distribution: 'microsoft'
```

Comment:
Just curious why this change was needed - is the zulu version not compatible with the 3DS flow?

Reply:
Yeah there was a random unit test failing and the Microsoft one resolved the issue. I figured it was worth a shot since it's backed by MS, it should have good support going forward.